src/NewsService.Business/Commands/News/CreateNewsCommand.cs

```diff
@@ -66,9 +73,9 @@ public class CreateNewsCommand : ICreateNewsCommand
 
       OperationResultResponse<Guid?> response = new();
 
-      _httpContextAccessor.HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
+      DbNews news = _mapper.Map(request);
 
-      response.Body = await _repository.CreateAsync(_mapper.Map(request));
+      response.Body = await _repository.CreateAsync(news);
```

Comment:
Why do you rewrite it? You do not use variable news from 76 line, there is no need create the news variable